hydromt_fiat/workflows/exposure_vector.py

```diff
@@ -1019,15 +1019,11 @@ def update_max_potential_damage(
             for c in updated_max_potential_damages.columns
             if "Max Potential Damage:" in c
         ]
-        updated_max_potential_damages.sort_values("Object ID", inplace=True)
-        self.exposure_db.sort_values("Object ID", inplace=True)
+        updated_max_potential_damages.set_index("Object ID", inplace=True)
+        self.exposure_db.set_index("Object ID", inplace=True, drop=False)
```

Comment:
This removes the Object ID as column and the test_update_max_potential_damage test fails due to mismatch in column numbers. 
Is it intended to have the Object ID column removed?